R/get_plan_doc_info.R

```diff
@@ -0,0 +1,36 @@
+#' Get planning doc info
+#'
+#' Get specific information for a workshop needed to set up the planning document.
+#'
+#' @param ws_dat workshop data: information read from Holy Excel sheet: dataframe with
+#' information about the workshop - used to set up the planning document
+#'
+#' @return planning doc information to be used in rendering the doc
+#'
+#' @export
+#'
+get_plan_doc_info <- function(ws_dat) {
```

Comment:
```suggestion
get_plan_doc_info <- function(info) {
```
consistent with `get_comms_doc_info`